### ISSUE TITLE
Emissao para cidade de Meridiano e Votuporanga/SP

### ISSUE THREAD

**Issue by murilomenegasso** [4success/nfse-campinas#2]
Boa noite amigo, consigo usar sua lib para emissao de NFSe para cidade de Meridiano e Votuporanga?

**Comment by renanwilliam** [comment]
Olá Murilo,

Essa lib deve funcionar com qualquer prefeitura com Padrão DSFNET.
Essas duas cidades me parecem usar o padrão ABRASF.

Acredito que você consiga reutilizar bastante coisa dessa lib mas seria necessário adaptar para esse outro padrão.

**Comment by renanwilliam** [comment]
Só atualizando, agora a lib se comunica no padrão ABRASF. Deve funcionar :)

**Comment by murilomenegasso** [comment]
Bom dia amigo, show.
Está servindo para qualquer município padrão Abrasf ? Faz um tempo que não
pego na NFSe, mas vou precisar voltar mexer

Em qui., 6 de fev. de 2025 09:03, Renan William ***@***.***>
escreveu:

**Comment by renanwilliam** [comment]
Sim, versão ABRASF **2.03**, que é a adotada pelo município de Campinas.

**Comment by murilomenegasso** [comment]
Show de bola mano.
Por pura coincidência vou começar a emitir NFSe para mim mesmo, ja testo
aqui em Votuporanga.

Obg... qualquer coisa que puder ajudar estamos ai..

Em qui., 6 de fev. de 2025 às 10:52, Renan William ***@***.***>
escreveu:
